common/src/main/java/gjum/minecraft/civ/snitchmod/common/Renderer.java

```diff
@@ -119,7 +118,7 @@ private static void renderSnitch(Snitch snitch) {
 		// inflate/deflate so the box face isn't obscured by adjacent blocks
 		final boolean playerInRange = range.contains(mc.player.position());
 		AABB rangeBox = playerInRange ? range.inflate(-.01) : range.inflate(.01);
-		AABB outlineBox = playerInRange ? range.inflate(-.05) : range.inflate(.05);
+		AABB outlineBox = playerInRange ? range.inflate(-.01) : range.inflate(.01);
```

Comment:
does this not result in z fighting?

Reply:
It didn't for me. It isn't really necessary though, it just always bothered me to have the box that much smaller than the lines.